lib/src/initializer.dart

```diff
@@ -69,20 +33,59 @@ class Initializer {
     required this.toolsMessage,
   });
 
+  /// Get a string representation of the current date in the following format
+  /// yyyy-MM-dd (2023-01-09)
   String get dateStamp {
-    return DateFormat('yyyy-MM-dd').format(DateTime.now());
+    return DateFormat('yyyy-MM-dd').format(clock.now());
```

Comment:
As with `config_handler.dart`, can we allow for direct injecting test-controlled clocks?